osquery/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(OSQUERY_KERNEL_BENCHMARKS "")
 # osquery core additional sources files not included with SDK (libosquery_additional).
 set(OSQUERY_ADDITIONAL_SOURCES "")
 set(OSQUERY_ADDITIONAL_LINKS "")
-set(OSQUERY_ADDITIONAL_TESTS "")
+set(OSQUERY_ADDITIONAL_TESTS "" tables/other/tests/augeas_tests.cpp)
```

Comment:
CLion added this automatically. Not sure if it's required for the augeas tests to run.